modxcms/revolution
# Settings grid adjustments for Extras support

### What does it do?
Re-introduced original id property on settings grid filter combos and added a couple small changes already developed for the much larger #16369.

### Why is it needed?
Provides a quick solution to a couple problems caused by the unintentional early implementation of #16089 into the 3.0.3 release, namely those related to Extras that  extend the settings grid (specifically @Jako's Agenda and CrossContextsSettings).

### How to test

- Ensure that the ACL grids in the Usergroup Permissions area (5 grids) and the Settings grid behave as they currently do in 3.0.3.
- Test the two Extras mentioned above to verify they behave as intended.

### Related issue(s)/PR(s)
Fulfills request made in the closed #16408
